calc/src/mechanics/gen2.ts

```diff
@@ -58,7 +58,7 @@ export function calculateGSC(attacker: Pokemon, defender: Pokemon, move: Move, f
     return result;
   }
 
-  const lv = attacker.level;
+  var lv = attacker.level;
```

Comment:
const is correct

Reply:
Const is not correct, Present alters the level of used in the calculation. If lv is a constant, it cannot be modified which prevents Present from having an accurate calculation,

Reply:
Than it should be ``let``

Reply:
Ah, right. Fixed.